XTS/wsat-jta-multi_hop/pom.xml

```diff
@@ -19,6 +19,10 @@
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
         <narayana.version>5.2.9.Final-SNAPSHOT</narayana.version>
         <version.org.wildfly.arquillian>1.0.1.Final</version.org.wildfly.arquillian>
+        <ipv6.server.jvm.args>-Djboss.bind.address=[::1] -Djboss.bind.address.management=[::1] -Djboss.bind.address.unsecure=[::1] -Djava.net.preferIPv4Stack=false -Djava.net.preferIPv6Addresses=true</ipv6.server.jvm.args>
+        <jvm.args.memory>-Xms64m -Xmx1024m -XX:MaxPermSize=512m</jvm.args.memory>
+        <jvm.args.debug></jvm.args.debug>
+        <jvm.args.other>-server</jvm.args.other>
```

Comment:
What does this do (I couldn't google the properties and didn't fancy digging out the XSD)